Mathlib/RingTheory/SimpleRing/Principal.lean

```diff
@@ -0,0 +1,30 @@
+/-
+Copyright (c) 2025 Yakov Pechersky. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yakov Pechersky
+-/
+
+import Mathlib.RingTheory.PrincipalIdealDomain
+import Mathlib.RingTheory.SimpleRing.Field
+import Mathlib.RingTheory.TwoSidedIdeal.Operations
+
+/-!
+# A simple ring is trivially a principal ideal d
+
+-/
+
+variable {R : Type*} [CommRing R] [IsSimpleRing R]
+
+instance : IsSimpleOrder (Ideal R) :=
+  ⟨fun I ↦ (eq_bot_or_eq_top I.toTwoSided).imp
+    -- could also be done through the `gc`?
+    (by simp [TwoSidedIdeal.ext_iff, Ideal.ext_iff])
+    (by simp [TwoSidedIdeal.ext_iff, Ideal.ext_iff])⟩
```

Comment:
Can you use [OrderIso.isSimpleOrder](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Order/Atoms.html#OrderIso.isSimpleOrder) with [TwoSidedIdeal.orderIsoIdeal](https://leanprover-community.github.io/mathlib4_docs/Mathlib/RingTheory/TwoSidedIdeal/Operations.html#TwoSidedIdeal.orderIsoIdeal)?